input/pages/artifact-repository-service.md

```diff
@@ -211,8 +216,30 @@ The _publish_ operation supports posting a new artifact with _active_ status. Th
 
 The _release_ operation supports updating the status of an existing _draft_ artifact to _active_. The operation sets the _date_ element of the resource and pins versions of all direct and transitive references. Child artifacts (i.e. artifacts that _compose_ the existing artifact) are also Released, recursively.
 
+The following parameters SHOULD be supported for the operation:
+
+* Instance level:
+    * **id**: The server-specific id of the artifact to be released.
+* Type level:
+    * **url**: The canonical url of the artifact to be released.
+    * **version**: The current version of the artifact to be released.
+    * **identifier**: A business identifier of the artifact to be released.
+    * **resource**: The resource type of artifact to be released.
+
 ##### Draft
-The _draft_ operation supports the creation of a new draft version of an existing artifact in _active_ status. This operation creates a new resource with the same contents as the existing artifact, but with a status of draft and no version. It is an error to create multiple drafts of the same artifact by URL.
+The _draft_ operation supports the creation of a new draft version of an existing artifact in _active_ status. This operation creates a new resource with the same contents as the existing artifact, but with a status of _draft_ and a pre-release label of `-draft` appended to the version.
+
+The following parameters SHOULD be supported for the draft operations:
+
+* **draftVersion**: The version of the artifact which is in review, i.e. the version under which it will be released
```

Comment:
$draft already has `version` , but this refers to the version which will be set on the new artifact. Use `draftVersion` to refer to the version which is being set and leave `version` as the type level parameter?

Reply:
i.e. `version` to search for the input and `draftVersion` is set on the output